tests/integration.rs

```diff
@@ -4129,52 +4121,40 @@ mod common_parallel {
             DIRECT_KERNEL_BOOT_CMDLINE,
         );
 
-        let r = std::panic::catch_unwind(|| {
-            curl_command(
-                &api_socket,
-                "PUT",
-                "http://localhost/api/v1/vm.create",
-                Some(&http_body),
-            );
+        assert!(simple_api_command(&mut socket, "PUT", "create", Some(&http_body)).is_ok());
```

Comment:
For some reason if command fails, shouldn't we assert in tests?. Curl command was also does assert in case of unsuccessful command operation.

Reply:
I mean there as well.
You mean something like this assert!(simple_api_full_command(&mut socket, "GET", "vmm.ping", None).unwrap());

Reply:
The `unwrap()` on `Err` would panic like `assert` would panic on `false`, check more here: https://doc.rust-lang.org/std/result/enum.Result.html#method.unwrap. 

Also, it is good to take some time to understand how `std::result::Result` work :https://doc.rust-lang.org/stable/std/result/enum.Result.html

Reply:
When I see documentation about it "Because this function may panic, its use is generally discouraged. Instead, prefer to use pattern matching and handle the [Err](vscode-file://vscode-app/snap/code/122/usr/share/code/resources/app/out/vs/code/electron-sandbox/workbench/workbench.html) case explicitly, or call [unwrap_or], [unwrap_or_else], or [unwrap_or_default]." 

Bit hesitated to use it in the beginning.